public/locales/en/common.json

```diff
@@ -253,5 +252,9 @@
   "sentEmailTo": "We have sent an email to",
   "toCheckEmail": "for you to verify and confirm the email address you provided.",
   "pleaseCheckYourEmail": "Please check your emails (Step 2 of 3)",
-  "unsubscribeConfirmation": "Please click the confirmation link in the email within 24 hours to finish unsubscribing."
+  "unsubscribeConfirmation": "Please click the confirmation link in the email within 24 hours to finish unsubscribing.",
+  "withdraw": "How to withdraw your information",
```

Comment:
The `withdraw` key was missed in the french file
![image](https://user-images.githubusercontent.com/34345435/126179889-40319ebf-8839-439d-8797-010701aa3ad6.png)